examples/Cargo.toml

```diff
@@ -0,0 +1,163 @@
+[package]
+name = "gfx-examples"
+version = "0.7.0"
+description = "GFX example application framework"
+homepage = "https://github.com/gfx-rs/gfx"
+keywords = ["graphics", "gamedev"]
+license = "Apache-2.0"
+authors = ["The Gfx-rs Developers"]
+documentation = "https://docs.rs/gfx_project"
+exclude = ["bors.toml", ".travis.yml"]
+workspace = ".."
+
+[features]
+default = ["gl"]
+mint = ["gfx/mint", "gfx_core/mint"]
+metal = ["gfx_device_metal", "gfx_window_metal", "gfx_device_metalll"]
+metal_argument_buffer = ["gfx_device_metalll/argument_buffer"]
+gl = ["gfx_device_gl", "gfx_window_glutin"]
+dx11 = ["gfx_device_dx11", "gfx_window_dxgi"]
+dx12 = ["gfx_device_dx12", "gfx_window_dxgi"]
+vulkan = ["gfx_device_vulkan", "gfx_device_vulkanll", "gfx_window_vulkan"]
+sdl = ["gfx_window_sdl"]
+serialize = ["gfx/serialize", "gfx_core/serialize"]
+headless = ["gfx_window_glutin/headless"]
+unstable = []
+
+[dependencies]
+log = "0.3"
+env_logger = "0.4"
+glutin = "0.9"
+winit = "0.7"
+
+cgmath = "0.15"
+gfx_gl = "0.3"
+rand = "0.3"
+genmesh = "0.5"
+noise = "0.4"
+image = "0.15"
+
+gfx = { path = "../src/render", version = "0.16" }
+gfx_core = { path = "../src/core", version = "0.7.1" }
```

Comment:
I was able to remove:
gfx_macros

I needed to keep all the rest to build the examples, since they all use one Cargo.toml file now. Unless I'm missing something, I don't see how we would remove any more dependencies listed in this file and still have the examples runnable.